lib/pf/condition/includes.pm

```diff
@@ -1,20 +1,31 @@
-package pf::triggerParser::dhcp_fingerprint;
+package pf::condition::includes;
 =head1 NAME
 
-pf::triggerParser::dhcp_fingerprint - Trigger for dhcp_fingerprint
+pf::condition::includes add documentation
 
 =cut
 
 =head1 DESCRIPTION
 
-pf::triggerParser::dhcp_fingerprint
+pf::condition::includes
 
 =cut
 
 use strict;
 use warnings;
-use Moo;
-extends 'pf::triggerParser';
+use Moose;
+extends qw(pf::condition);
+
+has value => (
+    is => 'ro',
+    required => 1,
+    isa  => 'Str',
+);
+
+sub match {
+    my ($self,$arg) = @_;
+    return $self->value ~~ $arg;
```

Comment:
This does not seem correct.
Why are you searching a string as if it is an array?